<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#bgWidget{
	background-color: rgb(61, 56, 70);
}

QLabel {
	color: rgb(255, 255, 255);
	font: 20pt &quot;Noto Mono&quot;;
}

QLabel#dialogName {
	font: 36pt &quot;Noto Mono&quot;;
}


QPushButton {
	border-radius: 20px;
	background-color: rgb(87, 227, 137);
	font: 14pt &quot;Noto Mono&quot;;
}

QPushButton:hover{
	background-color: rgb(46, 194, 126);
}</string>
  </property>
  <widget class="QWidget" name="bgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">



</string>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>500</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimum">
     <number>8</number>
    </property>
    <property name="maximum">
     <number>64</number>
    </property>
    <property name="value">
     <number>32</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="dialogName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>500</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Generate password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>590</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="generationError">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>681</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(237, 51, 59);
font: 9pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>270</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Password length:</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLength">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>32</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="smallLetters">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>abc</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="capitalLetters">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>ABC</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="digits">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="specialChars">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>!$*</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="iconsqrt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>passwordLength</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
